test/servico/cst24Test.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package servico;
+
+import auxiliar.Perfil;
+import datamapper.PopulateDB;
+import datamapper.exceptions.NonexistentEntityException;
+import java.text.ParseException;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import modelo.UsuarioModel;
+
+/*
+ *
+ * @author Mohamad
+ *
+ */
+
+public class cst24Test {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Class names should begin with an uppercase character](https://www.codacy.com/app/calebepb/substituicao/file/2769137731/issues/source?bid=3308597&fileBranchId=3353677#l31)